### DISCUSSION TITLE
Custom Sort

### DISCUSSION THREAD

**Discussion by smcguffee** [zim-desktop-wiki/zim-desktop-wiki#2155] [Ideas]
Hi, 
I just started trying out Zim, and I like it a lot.
It's much better than Tomboy for example because it handles links really well.
One issue I have though is that I cannot sort my notes according to where I want them without changing the contents of the title of the note. I would like to be able to move them around. Maybe related to this, the tabs move around depending on which note I'm in most recently. I would like to be able to count on a tab staying where it has been, so when I want to go to it, I can go to where the tab had been. It would also be helpful for focus if there were a way to close a tab.

**Comment by shawnhcorey** [comment]
On 2022-09-20 12:19, Sean Robert McGuffee wrote:
>
> Hi,
> I just started trying out Zim, and I like it a lot.
> It's much better than Tomboy for example because it handles links 
> really well.
> One issue I have though is that I cannot sort my notes according to 
> where I want them without changing the contents of the title of the 
> note. I would like to be able to move them around. Maybe related to 
> this, the tabs move around depending on which note I'm in most 
> recently. I would like to be able to count on a tab staying where it 
> has been, so when I want to go to it, I can go to where the tab had 
> been. It would also be helpful for focus if there were a way to close 
> a tab.
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/2155>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAA2CVDVDBPRBIPV6D5FAB3V7HPYTANCNFSM6AAAAAAQRHUFNY>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: 
> ***@***.***>
>
As a work around, number the titles. It doesn't look as good but it 
works. For example:

1.0 First Title

1.1 subtitle

1.2 subtitle

2.0 Second

3.0 Third
